Application/EdFi.Ods.AdminApi/Docker/pgsql/run.sh

```diff
@@ -25,4 +25,4 @@ if [[ -f /ssl/server.crt ]]; then
   update-ca-certificates
 fi
 
-dotnet EdFi.Ods.AdminApi.dll
+dotnet EdFi.Ods.Admin.Api.dll
```

Comment:
The name needs to be `AdminApi`, not `Admin.Api`, at Jason's request. The csproj file is `EdFi.Ods.AdminApi.csproj`, so the dll should already be `AdminApi`.

Reply:
@stephenfuqua 
This is temporary fix for work with 1.2.2 -> which is specified version on the pgsql.Docker file which is referenced from compose-build-binaries. Is that ok to update it ?

Reply:
That version number is no longer correct though. There is no reason to build version 1.2.2 from the `main` branch.

I meant to change that dockerfile to take an ARG instead of an ENV so that the value could be replaced by the build process. I don't know that I'll have time to work on that this week ☹️ . I'm trying to make the same change in the ODS/API right now so that we can have fresh prereleases of the ODS/API for use by Admin App and Admin API e2e tests.